templates/artist/artist_detail.html

```diff
@@ -22,7 +23,9 @@ <h1 class="header-font">{{ artist.name }}</h1>
     <div class="break">
         <hr>
     </div>
-    <img src="{{ artist.photo.img.url }}" />
+    {% thumbnail artist.photo.img "500" as thumb %}
+    <img src="{{ thumb.url }}" width="{{ thumb.width }}" height="{{ thumb.height }}" alt="{{ artist.name }}" />
```

Comment:
Perfect! Now you just need to indent the `<img />` tag so that it's obvious that it's an image of the thumbnail generated by the `{% thumbnail %}` tag.